delayed_spans_test.go

```diff
@@ -50,6 +51,14 @@ func TestPartiallyFlushDelayedSpans(t *testing.T) {
 
 	generateSomeTraffic(s, maxDelayedSpans)
 
+	// serverless agent should not be present for this test to pass.
+	// following check is added for debugging random failures in the unit tests of delayed spans
+	// TODO: remove it once the issue is resolved.
+	_, isURLPresent := os.LookupEnv("INSTANA_ENDPOINT_URL")
+	_, isKeyPresent := os.LookupEnv("INSTANA_AGENT_KEY")
+	assert.Equal(t, false, isURLPresent)
```

Comment:
My first thought was not to print the credentials, as assert would print the actual values. This makes sense since we do not have any credentials in the GitHub actions. Also, I have removed the check for `INSTANA_AGENT_KEY` since it is irrelevant in this case.